src/api/api.ts

```diff
@@ -36,15 +37,18 @@ const requestSigning = (
   });
 };
 
-const handleValidRequest = (request: JsonRPCRequest, webContents: WebContents) => {
+const handleValidRequest = async (request: JsonRPCRequest, webContents: WebContents) => {
   switch (request.method) {
     case SUPPORTED_METHODS.SIGN_TRANSACTION:
       return requestSigning(request, webContents);
-    // @todo Actual account handling
+    // @todo Dont expose all accounts at once?
+    // @todo Decide whether to fetch directly from store?
     case SUPPORTED_METHODS.ACCOUNTS:
       return toJsonRpcResponse({
         id: request.id,
-        result: ['0x82D69476357A03415E92B5780C89e5E9e972Ce75']
+        result: Object.values(await getAccounts())
+          .filter((a) => a.address)
```

Comment:
Is this necessary? In what situation would `address` be `undefined` or `null`?

Reply:
It seems that some metadata is stored by `persist-redux`, maybe I should filter it out some other way. Hmm